src/content/it/2020/resource-hints.md

```diff
@@ -0,0 +1,366 @@
+
+### Best practices
+
+L'uso delle resource hints a volte può creare confusione, quindi esaminiamo alcune veloci best practice da seguire in base all'audit automatizzato di Lighthouse.
+
+Per implementare in modo sicuro `dns-prefetch` e `preconnect` assicurati di averli in tag di link separati.
+
+```html
+<link rel="preconnect" href="http://example.com">
+<link rel="dns-prefetch" href="http://example.com">
+```
+
+L'implementazione di un fallback `dns-prefetch` nello stesso tag `<link>` causa un [bug](https://bugs.webkit.org/show_bug.cgi?id=197010) in Safari che annulla la richiesta `preconnect`. Quasi il 2% delle pagine (~ 40k) ha segnalato il problema sia di `preconnect` che di `dns-prefetch` in una singola risorsa.
+
+Nel caso dell'audit "[Preconnect to required origins](https://web.dev/uses-rel-preconnect/)", abbiamo visto che solo il 19,67% delle pagine ha superato il test, creando una grande opportunità per migliaia di siti web per iniziare a utilizzare `preconnect` o `dns-prefetch` per stabilire connessioni iniziali a importanti origins di terze parti.
+
+{{ figure_markup(
+  caption="La percentuale di pagine che superano l'audit Lighthouse `preconnect`.",
+  content="19.67%",
+  classes="big-number",
+  sheets_gid="152449420",
+  sql_file="lighthouse_preconnect.sql"
+) }}
+
+Infine, l'esecuzione dell'audit "[Preload key requests](https://web.dev/uses-rel-preload/)" di Lighthouse ha portato l'84,6% delle pagine a superare il test. Se stai cercando di usare `preload` per la prima volta, ricorda che i fonts e gli script critici sono un buon punto di partenza.
+
+{# TODO(authors/reviewers) - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532291496 #}
+
+### Native Lazy Loading
+
+Ora festeggiamo il primo anno dell'API [Native Lazy Loading](https://addyosmani.com/blog/lazy-loading/), che al momento della pubblicazione ha già superato il [72%](https://caniuse.com/loading-lazy-attr) supporto browser. Questa nuova API può essere utilizzata per posticipare il caricamento di iframe e immagini sulla pagina finché l'utente non scorre accanto a essi. Ciò può ridurre l'utilizzo dei dati, l'utilizzo della memoria e velocizzare i contenuti. Attivare il lazy load è semplice come aggiungere `loading = lazy` sugli elementi `<iframe>` o `<img>`.
+
+La percentuale di pagine che utilizzano il Native Lazy Loading.
+
+{{ figure_markup(
+  caption="La percentuale di pagine che utilizzano il native lazy loading.",
+  content="4.02%",
+  classes="big-number",
+  sheets_gid="2039808014",
+  sql_file="native_lazy_loading_attrs.sql"
+) }}
+
+{# TODO(authors/reviewers) - revisit this figure - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532292106 #}
+
```

Comment:
Can remove this (also from English if you get a chance!):

```suggestion
```

Reply:
@chefleo / @Zizzamia did you see this? Hidden under a "Load more..." so you might have missed it.